mindsdb/api/http/initialize.py

```diff
@@ -16,8 +24,87 @@ class Swagger_Api(Api):
     def specs_url(self):
         return url_for(self.endpoint("specs"), _external=False)
 
+
+def initialize_static():
+    this_file_path = os.path.abspath(inspect.getfile(inspect.currentframe()))
+    static_path = Path(this_file_path).parent.joinpath('static/')
```

Comment:
In the future, remember to move this to the `var` dir where we store predictors, datasources ... etc.